jax/numpy/lax_numpy.py

```diff
@@ -1465,18 +1472,31 @@ def _pad(array, pad_width, mode, constant_values):
       widths[i] = (0, pad_width[i, 1], 0)
       array = lax.pad(array, constant_values[i, 1], widths)
     return array
-  elif mode in ("symmetric", "reflect", "wrap"):
+  elif mode == "wrap":
+    for i in xrange(nd):
```

Comment:
Shouldn't this be `range`, or won't you need `six.moves` import of `xrange`?

Reply:
I just adapted this from what you already had. It looks like this `xrange` is already `six.moves.xrange`.